packages/console/src/pages/ApplicationDetails/index.tsx

```diff
@@ -47,7 +47,7 @@ const mapToUriFormatArrays = (value?: string[]) =>
   value?.filter(Boolean).map((uri) => decodeURIComponent(uri));
 
 const mapToUriOriginFormatArrays = (value?: string[]) =>
-  value?.filter(Boolean).map((uri) => decodeURIComponent(new URL(uri).origin));
+  value?.filter(Boolean).map((uri) => decodeURIComponent(uri.replace(/\/*$/, '')));
```

Comment:
nit: seems the logic has been changed, should we also rename the function?

Reply:
the original naming is tricky and it plays as the same purpose i think. not sure about it

Reply:
discussed offline